api/v1alpha1/hostedcluster_types.go

```diff
@@ -195,7 +195,9 @@ type ClusterVersionStatus struct {
 // +kubebuilder:subresource:status
 // +kubebuilder:printcolumn:name="Version",type="string",JSONPath=".status.version.history[?(@.state==\"Completed\")].version",description="Version"
 // +kubebuilder:printcolumn:name="KubeConfig",type="string",JSONPath=".status.kubeconfig.name",description="KubeConfig Secret"
+// +kubebuilder:printcolumn:name="Progress",type="string",JSONPath=".status.version.history[0].state",description="Progress"
 // +kubebuilder:printcolumn:name="Available",type="string",JSONPath=".status.conditions[?(@.type==\"Available\")].status",description="Available"
+// +kubebuilder:printcolumn:name="Reason",type="string",JSONPath=".status.conditions[0].reason",description="Reason"
```

Comment:
Can we please filter by type as above rather than using arbitrary `0`?

Reply:
@enxebre - does this also apply to the `Progress` column? should I find a field to filter on? Maybe

```
.status.version.history[?(@.verified==\"false\")].state
```

Reply:
verified=false can't no be relied upon as a discriminator.
How about `.status.version.history[?(@.state==\"Partial\")].state` ?

Reply:
@enxebre  - What if I just do this - since the state is either `Partial`  or `Completed` - can i just filter on `@.state!=\"\"` ?

that give me this:
```
$ oc get hostedclusters
NAME   VERSION      KUBECONFIG             PROGRESS    AVAILABLE   REASON
sc1    4.8.0-fc.0   sc1-admin-kubeconfig   Completed   True        HostedClusterIsAvailable
```

Reply:
sgtm! Can you please squash commits?